app/services/metrc_service/plant/discard.rb

```diff
@@ -27,7 +27,7 @@ def discard
         return @obj if @obj
 
         id = @relationships.dig('action_result', 'data', 'id')&.to_i || @completion_id
-        @obj = batch.discard(id)
+        @obj = get_completion(id)
```

Comment:
For some reason, the portal returns a `404` if we use the discard endpoint. So, we use this one instead.